data.sql

```diff
@@ -3,3 +3,19 @@
 INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
 INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animal(id, animalname, animal_date_of_birth,
+animal_escape_attempts, neutered, animal_weight_kg) VALUES (
+    1, 'Agumon', '2020-02-03', 0, FALSE, 10.23
```

Comment:
- [ ] Kindly change this 'neutered' column for Agumon to 'true' as it is the requirement.